standards/secrets-management.md

```diff
@@ -0,0 +1,30 @@
+---
+expires: 2019-03-15
+---
+# Secrets management
+
+'Secrets' is defined here as a sensitive bit of information that should be kept private but with a technical system or user focus, such as, a password, OAuth token or SSH private key.
+
+'Secrets' here is **not** referring to the `SECRET` classification.
+
+## The base principle
+
+All secrets **must** be adequately protected from a loss of confidentiality or integrity. Secrets, much like other confidential data, must only be accessible or otherwise open to influence from intended authorised parties.
```

Comment:
"otherwise open to influence from" reads strangely.

Reply:
Language updated. PR inbound.